.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Thanks for raising an issue! To help us help you, if you've found a bug please consider the following:
+
+* If you can demonstrate the bug using the interactive [rollupjs.org](http://rollupjs.org) demo, please do.
```

Comment:
Yeah, wondered about that. Initially it wasn't commented, and I thought the markdown would be helpful as people could switch to preview mode and use the links etc, but looking around it seems people do actually treat it as a template, so I added the `<!--` and `-->`. I guess it's just the links that don't really make sense? Other stuff like bullets and bold kind of work in both contexts